### ISSUE TITLE
My node is so slowly syncs!!!

### ISSUE THREAD

**Issue by gitloverr** [bnb-chain/bsc#980]
Hello everyone,

 my node so slowly sync after last release.

my starter parameter 

`./geth --config xxxconfig.toml --datadir xxx --metrics --pprof --pprof.addr 0.0.0.0 --maxpeers 1000 --cache 18000 --http --http.addr 0.0.0.0 --http.port xxx --http.corsdomain "*" --ws --ws.origins="*" --ws.addr 0.0.0.0 --allow-insecure-unlock --diffsync --rpc.allow-unprotected-txs --txlookuplimit 0  --snapshot=false --rpcvhosts=*`

what is the problem?

what can I do for solve this subject?

thank you

**Comment by cosinlink** [comment]
1. remove `--diffsync`
2. use separate node to run bsc, refer to : https://github.com/bnb-chain/bsc/pull/926,
 - use develop branch, compile to geth
 - `./geth --config ./config.toml --datadir ./node --syncmode full --cache 5000 --tries-verify-mode none`

**Comment by gitloverr** [comment]
hi,

if I restart the node it is catching... but there is something I noticed: if I start send requests to the node I starting to lose sync...

this is ridiculous... how do I solve this problem?

thank you

**Comment by cosinlink** [comment]
> 

could you provide more details or screenshot?

**Comment by gitloverr** [comment]
hi,
my machine specs are as follows:
Disk: SSD
CPU: 48 Core
RAM: 192GB
IOPS: 10K

If I send the requests over as WS or HTTP , the node falling behind... if I stop sending request I don't have sync problem. I guess, this is happening due to request limits. this happened after the 1.1.11 release...   

also my machine cpu & ram load that's like: 

<img width="892" alt="Image_CPU_RAM" src="https://user-images.githubusercontent.com/80384301/178279328-93ed0181-0f67-4cbe-9c80-1cb1662d6dcb.png">

**Comment by cosinlink** [comment]
Sorry for missing that, have you fixed this issue ?

**Comment by xiyuewang3** [comment]
Me too

**Comment by odannis** [comment]
My node has the same problem. It was fine for months, and one week ago it started to falling behind the current block (10-60 min behind). I updated geth-bsc to the latest release, started for a fresh snapshot, but the difficulty persisted.
I moved to erigon for bsc, and it is working well except for some rpc request..


 I have several other nodes (fantom, eth, avax) running on the same computer and everything work well.
